jest.json

```
@@ -18,14 +18,14 @@
         "jest/no-restricted-matchers": [
           "error",
           {
-            "toMatchSnapshot": null,
-            "toMatchInlineSnapshot": null,
-            "toThrowErrorMatchingSnapshot": null,
-            "toThrowErrorMatchingInlineSnapshot": null,
-
-            "toBeTruthy": null,
-            "toBeFalsy": null,
-            "toBeDefined": null
+            "toMatchSnapshot": "Use more targeted assertions instead of a snapshot. See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#snapshots",
+            "toMatchInlineSnapshot": "Use more targeted assertions instead of a snapshot. See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#snapshots",
+            "toThrowErrorMatchingSnapshot": "Use more targeted assertions instead of a snapshot. See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#snapshots",
+            "toThrowErrorMatchingInlineSnapshot": "Use more targeted assertions instead of a snapshot. See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#snapshots",
```

```suggestion
            "toMatchSnapshot": "Use more targeted assertions instead of a snapshot",
            "toMatchInlineSnapshot": "Use more targeted assertions instead of a snapshot",
            "toThrowErrorMatchingSnapshot": "Use more targeted assertions instead of a snapshot",
            "toThrowErrorMatchingInlineSnapshot": "Use more targeted assertions instead of a snapshot",
```

That GitLab article doesn't explain why snapshots are forbidden.

---

Maybe we should add a link to https://github.com/EvgenyOrekhov/eslint-config-hardcore/issues/465#issuecomment-1058052048?